.github/workflows/php.yml

```diff
@@ -4,10 +4,15 @@ on: [push, pull_request]
 
 jobs:
   test:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.0
     steps:
       - uses: actions/checkout@v2
 
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 7.4
+
```

Comment:
I guess you referred to that https://github.com/marketplace/actions/setup-php-action for this change, right @dgwatkins?

Reply:
Yes, thats right @strategio 

But the action didn't run, its been queued for 15h

I will have to look for a different approach (update PHPUnit maybe?)

Reply:
I was using the wrong Ubuntu image.
Now it passes.

I made an extra commit so I will need your re-approval @strategio